src/entities/contract-negotiation.ts

```diff
@@ -1,28 +1,39 @@
-import { Policy } from "./policy";
+import { JsonLdId, JsonLdValue } from "./jsonld";
 
 export interface ContractNegotiationRequest {
   connectorAddress: string;
   connectorId: string;
+  providerId: string;
   offer: {
     offerId: string;
     assetId: string;
-    policy: Policy;
+    policy: any;
```

Comment:
Can we specialise this type?

Reply:
nope, or, better, not yet, I'm working on better handling of types, in the future it will be, but currently it brings into problems, I'd prefer to fix them when the situation will be more stable.